ci_framework/roles/devscripts/tasks/sub_tasks/15_user.yml

```diff
@@ -34,24 +33,8 @@
 
 - name: Generate SSH keys for accessing OCP cluster
   community.crypto.openssh_keypair:
+    comment: "cifmw generated key for ocp access"
     path: >-
       {{ cifmw_devscripts_artifacts_dir }}/{{ cifmw_devscripts_user }}_ed25519
     type: "ed25519"
   register: cifmw_devscripts_ssh_key
-
-- name: Enable password-less access
-  become: true
-  ansible.posix.authorized_key:
-    user: "{{ cifmw_devscripts_user }}"
-    key: "{{ cifmw_devscripts_ssh_key.public_key }}"
-    state: present
-
-- name: Verify sudoers privileges
-  become: true
-  ansible.builtin.copy:
-    dest: "/etc/sudoers.d/{{ cifmw_devscripts_user }}"
-    owner: root
-    group: root
-    mode: "0640"
-    content: |
-      {{ cifmw_devscripts_user }}    ALL=(ALL)    NOPASSWD: ALL
```

Comment:
we actually need that passwordless sudo for dev-scripts to properly run. Else we face:
```
[.... endless error trace....]
sudo mkdir -p /opt/devscripts/ironic/html/images", "sudo: timed out reading password", "sudo: a password is required
```

Reply:
Sorry about that... I assumed it would be present. I have added this task back in the next commit.